openedx/openedx-platform
# chore: remove paver update_db task

## Description
As part of the effort to deprecate paver overall hence the `update_db` task is being removed here.
This change would break any references to the paver `update_db` task.
Effort has been made to remove all refrences that can be found in other repos and would be linked to this PR appropiately.
Additional work to convert to an alternative `make migrate` has been done as part of https://github.com/openedx/devstack/issues/1085 and any left over work would be done and linked to this PR appropiately.


## Supporting information
This PR fixes https://github.com/openedx/edx-platform/issues/32683 and https://github.com/openedx/edx-platform/issues/32252


## Testing instructions
Before this change, you could run `paver update_db` in the lms.
After this change, on running `paver update_db` you should get an error message like `Build failed: Unknown task: update_db`
There are no references to this task in the docs.


Fixes: https://github.com/openedx/edx-platform/issues/32683 and https://github.com/openedx/edx-platform/issues/32252
Private-ref: https://tasks.opencraft.com/browse/BB-7705
